### PR TITLE
Add recipe for kdtree

### PR DESCRIPTION
**Pull request by lincrosenbach**

### CODE REVIEW

**Comment by nicoddemus** [comment]
@ocefpaf @igortg @marcelotrevisani @conda-forge/staged-recipes I believe this is ready for review 👍

**Comment by marcelotrevisani** [comment]
LGTM! Thanks for adding this! :)

File: recipes/kdtree/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  license: ISC (ISCL)
+  license_family: OTHER
+  license_file: LICENSE
+  summary: A Python implementation of a kd-tree
+  doc_url: https://python-kdtree.readthedocs.org/
+  dev_url: https://github.com/stefankoegl/kdtree
+
+extra:
+  recipe-maintainers:
+    - lincrosenbach
```
Comment (nicoddemus) [inline recipes/kdtree/meta.yaml]: @lincrosenbach could you please add me as a maintainer of this feedstock when it gets created? Thanks!

Comment (lincrosenbach) [reply recipes/kdtree/meta.yaml]: Sure!

**Pull request merged**